subsys/bluetooth/mesh/light_hsl_srv.c

```diff
@@ -411,9 +411,7 @@ static ssize_t scene_store(struct bt_mesh_model *model, uint8_t data[])
 	}
 
 	srv->lightness->handlers->light_get(srv->lightness, NULL, &status);
-	sys_put_le16(status.remaining_time ? light_to_repr(status.target, ACTUAL) :
-					     status.current,
-		     &data[0]);
+	sys_put_le16(status.remaining_time ? to_actual(status.target) : status.current, &data[0]);
```

Comment:
It looks like `status.current` should also be converted.

Reply:
Good catch! Looks like this was an existing bug